### PR TITLE
[backport] Add build progress monitoring tool

### PR DESCRIPTION
**Pull request by emcastillo**
Backport of #162 by @kmaehashi

### CODE REVIEW

**Comment by emcastillo** [comment]
[automatic post] Jenkins, test this please.

**Comment by emcastillo** [comment]
Conflicted, review please

File: .pfnci/wheel-linux/submit.sh [hunk 1]
```diff
@@ -1,16 +1,36 @@
-#!/bin/sh -uex
+#!/bin/bash
 
 # Submit as FlexCI jobs
```
Comment (kmaehashi) [inline .pfnci/wheel-linux/submit.sh:4]:
```suggestion

set -u

```

File: .pfnci/wheel-windows/submit.sh [hunk 2]
```diff
@@ -1,15 +1,35 @@
-#!/bin/sh -uex
+#!/bin/bash
 
 # Submit as FlexCI jobs
```
Comment (kmaehashi) [inline .pfnci/wheel-windows/submit.sh:4]:
```suggestion

set -u

```

**Review by kmaehashi — CHANGES_REQUESTED** [review]

**Review by kmaehashi — APPROVED** [review]

**Comment by kmaehashi** [comment]
Merging as these changes are not covered by tests

**Pull request merged**